### ISSUE TITLE
DataParallel model stuck on loss.backward()

### ISSUE THREAD

**Issue by nirandiw** [cv-core/MIT-Driverless-CV-TrainingInfra#5]
I have tried running the yolo_tutorial.ipynb code on 4 GPUs. For some reason, it gets stuck in the `run_epoch` method. Further analysis showed that get's stuck at the `losses[0].sum().backward()`. I am using pytorch 1.3.1. 

Has anyone else tested this code on multiple GPUs and run in to the same issue?

I can run the code in a single GPU without an issue.

**Comment by Sibozhu** [comment]
Hi @nirandiw Thanks for your interest. Sorry we haven't applied multi GPU training on our codebase. We are considering include them in the next version.
